go.mod

```diff
@@ -11,13 +11,14 @@ require (
 	github.com/prometheus/client_golang v1.12.1
 	github.com/spf13/pflag v1.0.5
 	github.com/stolostron/go-log-utils v0.1.1
-	github.com/stolostron/go-template-utils/v2 v2.5.1
+	github.com/stolostron/go-template-utils/v2 v2.6.0
 	github.com/stretchr/testify v1.7.0
 	k8s.io/api v0.23.9
 	k8s.io/apimachinery v0.23.9
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/klog v1.0.0
 	k8s.io/klog/v2 v2.70.1
+	k8s.io/kube-openapi v0.0.0-20220124234850-424119656bbf
```

Comment:
It's curious this became a direct dependency...maybe it should have been previously...

Reply:
Yeah, I don't know why it wasn't before but it's directly imported.